### DISCUSSION TITLE
How to add a route

### DISCUSSION THREAD

**Discussion by xcyeye** [vuepress/core#478] [Q&A] [answered]
I want to add a route to one of my components like a vue project. I would like to ask, besides the addRoute() method, is there any other way to add a route? I read the documentation. Is it in the node stage of the theme? Just export a route

**Comment by Mister-Hope** [comment, marked as answer]
You should check thi:

- https://v2.vuepress.vuejs.org/guide/migration.html#plugin-api-change
- https://v2.vuepress.vuejs.org/reference/node-api.html#createpage

![image](https://user-images.githubusercontent.com/33315834/156138608-364a63e0-3028-445b-ba5b-41851e9b06f7.png)
